enhancements/microshift/microshift-supportability-tools.md

```diff
@@ -0,0 +1,384 @@
+---
+title: MicroShift supportability tools
+authors:
+  - pacevedom
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@pmtk, MicroShift contributor"
+  - "@TurboTurtle, SOS contributor"
+approvers:
+  - "@dhellmann"
+api-approvers:
+  - None
+creation-date: 2023-01-17
+last-updated: 2023-01-17
+tracking-link:
+  - https://issues.redhat.com/browse/USHIFT-389
+see-also:
+  - "/enhancements/oc/must-gather.md"
+  - "https://sos.readthedocs.org/"
```

Comment:
We can already benefit from `sos` report generation. See https://github.com/openshift/microshift/blob/main/docs/debugging_tips.md#generating-an-sos-report

Do we want to add it here as a reference?